src/main/java/mate/academy/hibernate/relations/Main.java

```diff
@@ -16,17 +22,20 @@
         SessionFactory sessionFactory = HibernateUtil.getSessionFactory();
 
         Country usa = new Country("USA");
-        CountryService countryService = null; // TODO: initialize this instance
+        CountryService countryService = new CountryServiceImpl(
+                new CountryDaoImpl(sessionFactory)); // TODO: initialize this instance
         countryService.add(usa);
 
         Actor vinDiesel = new Actor("Vin Diesel");
         vinDiesel.setCountry(usa);
-        ActorService actorService = null; // TODO: initialize this instance
+        ActorService actorService = new ActorServiceImpl(
+                new ActorDaoImpl(sessionFactory)); // TODO: initialize this instance
         actorService.add(vinDiesel);
 
         Movie fastAndFurious = new Movie("Fast and Furious");
         fastAndFurious.setActors(List.of(vinDiesel));
-        MovieService movieService = null; // TODO: initialize this instance
+        MovieService movieService = new MovieServiceImpl(
+                new MovieDaoImpl(sessionFactory)); // TODO: initialize this instance
```

Comment:
The `MovieService` instance is being initialized correctly with `MovieServiceImpl` and `MovieDaoImpl`. However, the TODO comment suggests that this initialization needs to be verified or completed. Ensure that the `MovieServiceImpl` constructor is correctly implemented to accept a `MovieDaoImpl` instance.